common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -331,7 +329,7 @@ def _update_position(self, context, number_of_display_items):
             self.position = 1
         elif context.get('requested_child') == 'last':
             self.position = number_of_display_items or 1
-        elif self.position is None or self.position > number_of_display_items:
+        elif self.position is None or self.position > number_of_display_items or context.get('efischer_silly_var'):
```

Comment:
This is the other half of my illustration commit. Note that the code in this block sets `self.position` to 3 as a part of `test_student_view_last_child`, and it never gets reset to the initial untouched state assumed by `test_render_student_view`, instead returning `{'position': 3}` when asked to render with no additional context.